WPMedia/fusion-news-theme-blocks
# THEMES-1240: Logical properties on header nav chain block

## Description
This PR converts the relevant CSS properties to the logical equivalent. There should be no visual differences with the site set  to left-to-right text direction, and horizontal top to bottom writing mode.

## Jira Ticket

- [THEMES-1240](https://arcpublishing.atlassian.net/browse/THEMES-1240)

## Acceptance Criteria

1. The above logical properties are applied and tested on the header nav chain.
2. Storybook is updated with RTL. 

## Test Steps

1. Checkout this branch `git checkout THEMES-1240`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/header-nav-chain-block`
4. Inspect the nav chain and see that it is using logical properties in the correct places.
5. It should match the designs in Figma.
6. Set `"textDirection": "rtl"` in the `"siteProperties"` object of the `blocks.json`.
7. Restart PageBuilder and verify that the styling adapts to the new text direction.

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1240]: https://arcpublishing.atlassian.net/browse/THEMES-1240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
